mlir/include/mlir/Dialect/MIOpen/Generator/Conv2dGenerator.h

```diff
@@ -17,14 +17,20 @@
 #include "mlir/IR/BuiltinOps.h"
 #include "mlir/Support/LogicalResult.h"
 
+#include "llvm/ADT/Optional.h"
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringRef.h"
+
 namespace mlir {
+
 class Conv2dGenerator {
 public:
+  enum Op { Fwd, Dummy, BwdData, BwdWeight };
```

Comment:
Could you think of a way to consolidate this enum with another, use a separate header if necessary.

https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/blob/2c09afa99dca7bcac3e2030cd84b80bf489af663/mlir/include/mlir/Dialect/MIOpen/MIOpenOps.h#L28

Reply:
I didn't know we had that - I'll try and consolidate once @whchung gets back to us about the dummy op

Reply:
Let’s remove conv2d_dummy. That was introduced to showcase how we can split one convolution into multiple ones.